src/tests/conftest.py

```diff
@@ -1,16 +1,29 @@
+from logging import getLogger, FileHandler
 from typing import List
 
 import pytest
 from _pytest.nodes import Item
 
 import consts
 from assisted_test_infra.test_infra import utils
-from service_client import log
+from service_client import log, SensitiveFormatter
 from service_client.client_validator import verify_client_version
 from tests.config import global_variables
 
 assert global_variables.pull_secret is not None, "Missing pull secret"
 
+@pytest.fixture(scope="session", autouse=True)
+def setup_worker_log(worker_id):
+    def configure_logger(logger):
+        fmt = SensitiveFormatter("%(asctime)s - %(process)d - %(name)s - %(levelname)s - %(message)s")
+        fh = FileHandler("test_infra_{worker_id}.log")
```

Comment:
what is worker id?

Reply:
Why we can't use test name for file name? I just not sure this worker_id somehow will help. We will have 100 log files in regression, how we should know wich one to open? 
Lets create logs folder and add it to .gitignore

Reply:
To my understanding, the biggest problem was that logs from different parallel executors were mixed in single file, so it was a problem to track a source of a problem. Currently there would be single continuous log per executor. 

Not sure if per-test logging is possible, taking into account that fixtures can be run before/after test. I'll look into it.